src/pages/components/map-related/MapLayers/BusToolTip.tsx

```diff
@@ -94,12 +95,22 @@ export function BusToolTip({ position, icon }: BusToolTipProps) {
         {showJson ? t('hide_document') : t('show_document')}
       </Button>
       {showJson && (
-        <pre>
-          {JSON.stringify(position, null, 2)}
-          <br />
-          {siriRide && JSON.stringify(siriRide, null, 2)}
-        </pre>
+        <div onClick={(e) => e.stopPropagation()}>
+          <CustomTreeView<Point> id={position.point?.id + ''} data={position} name={t('line')} />
+          {siriRide?.gtfsRideId && (
+            <CustomTreeView<SiriRideWithRelatedPydanticModel | undefined>
+              id={siriRide?.gtfsRideId + ''}
+              data={siriRide}
+              name={t('drive_direction')}
+            />
+          )}
+        </div>
+        // <pre>
+        //   {JSON.stringify(position, null, 2)}
+        //   <br />
+        //   {siriRide && JSON.stringify(siriRide, null, 2)}
+        // </pre>
```

Comment:
should it be removed?

Reply:
Yes!

That was the previous display.
I removed it just now.